rust/libnewsboat/tests/env.rs

```diff
@@ -18,3 +19,13 @@ fn t_get_default_browser() {
     env::set_var(&key, &opera);
     assert_eq!(utils::get_default_browser(), opera);
 }
+
```

Comment:
Please put this test into a separate file, named after the function it's testing. This way, tests won't ever interfere with other. (We don't have any interference here, but imagine if `get_default_browser` started checking the `HOME` variable too.)

You can rename this `env.rs` to `get_default_browser.rs`, too.